docs/deploy_daos_cluster_example.md

```diff
@@ -15,33 +15,6 @@ Deployment tasks described in these instructions:
 
 ## Clone the repository
 
----
```

Comment:
This entire explanation was not necessary. The user should just follow the instructions and clone the the repo to ~/google-cloud-daos.